src/ol/events/ErrorEvent.js

```diff
@@ -0,0 +1,25 @@
+/**
```

Comment:
This was a shared event between MapboxVector and STAC, but now that MapboxVector was removed I could also keep it outside of the events folder and copy it into stacUtils.js or STAC.js. Whatever you prefer.